edx/learner-pathway-progress
# ENT-6234: Removed default attribute from the learner_pathway_uuid.

**Description:** 
This PR removes `default` attribute from `LearnerPathwayProgress.learner_pathway_uuid` field. This field must be provided at the time of instance creation.

**JIRA:** [ENT-6234](https://2u-internal.atlassian.net/browse/ENT-6234)

**Dependencies:** No dependencies, however this ticket blocks the learner pathway progress page deployment. 

**Merge deadline:** Need to merge this soon to unblock other tickets that are blocked on this in the current sprint.

**Installation instructions:** List any non-trivial installation 
instructions.

**Testing instructions:**

1. Open page django admin page for learner pathway progress model at `/admin/learner_pathway_progress/learnerpathwayprogress/`
2. Make sure there is no button to add a new record 
4. User should be able to view/edit existing instances.

**Reviewers:**


**Merge checklist:**

**Post merge:**
      finished.
